### PR TITLE
Update UserGuide.md for Track2Gather

### PR DESCRIPTION
**Pull request by wei2912**
This PR updates the UserGuide as per the draft written in our previous meeting.

### CODE REVIEW

**Comment by wei2912** [comment]
This PR fixes https://github.com/AY2122S1-CS2103-W14-2/tp/issues/7.

File: docs/UserGuide.md [hunk 1]
```diff
@@ -23,18 +23,18 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
...
-   * **`add`**`n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : Adds a contact named `John Doe` to the Address Book.
+    *  **`sort`** : Sorts the contacts list, then displays the sorted contacts list.
 
-   * **`delete`**`3` : Deletes the 3rd contact shown in the current list.
+    *  **`find`** : Finds a person by name.
 
-   * **`clear`** : Deletes all contacts.
+    *  **`delete`** : Deletes a person at the specified index.
 
-   * **`exit`** : Exits the app.
```
Comment (wei2912) [inline docs/UserGuide.md:35]: Add back this command

File: docs/UserGuide.md [hunk 2]
```diff
@@ -23,18 +23,18 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
...
-   * **`list`** : Lists all contacts.
+    *  **`add`** : Adds a person to the address book.
 
-   * **`add`**`n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : Adds a contact named `John Doe` to the Address Book.
+    *  **`sort`** : Sorts the contacts list, then displays the sorted contacts list.
 
-   * **`delete`**`3` : Deletes the 3rd contact shown in the current list.
+    *  **`find`** : Finds a person by name.
 
-   * **`clear`** : Deletes all contacts.
```
Comment (wei2912) [inline docs/UserGuide.md:33]: Add back this command

File: docs/UserGuide.md [hunk 3]
```diff
@@ -64,129 +64,74 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
...
+### Adding a person: `add` [coming soon]
 
 Adds a person to the address book.
 
-Format: `add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]…​`
-
-<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
-A person can have any number of tags (including 0)
-</div>
+Format: `add n/NAME p/PHONE_NUMBER ha/HOME_ADDRESS e/EMAIL [wa/WORK_ADDRESS] [qa/QUARANTINE_ADDRESS] [as/ADD_SHN_PERIOD] [cn/ADD_CASE_NUMBER [kn/NEXT_OF_KIN_NAME] [kp/NEXT_OF_KIN_PHONE] [ka/NEXT_OF_KIN_ADDRESS]`
```
Comment (wei2912) [inline docs/UserGuide.md:79]: Missing `]`

**Review by luffingluffy — APPROVED** [review]
LGTM!

File: docs/UserGuide.md [hunk 4]
```diff
@@ -64,129 +64,74 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
...
-    specifying any tags after it.
+* The flags are optional, contacts will be sorted by name as a default.
+* The sort will be stable, following the given order of flags if any.
 
-Examples:
-*  `edit 1 p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@example.com` respectively.
-*  `edit 2 n/Betsy Crower t/` Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
-
-### Locating persons by name: `find`
+### Finding contacts by name: `find` [coming soon]
```
Comment (wei2912) [inline docs/UserGuide.md:96]: Restore back to AB3 version

File: docs/UserGuide.md [hunk 5]
```diff
@@ -64,129 +64,74 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
...
-
-Format: `clear`
-
-### Exiting the program : `exit`
-
-Exits the program.
-
-Format: `exit`
-
-### Saving the data
```
Comment (wei2912) [inline docs/UserGuide.md:157]: Add back this section

File: docs/UserGuide.md [hunk 6]
```diff
@@ -64,129 +64,74 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
...
-
-AddressBook data are saved in the hard disk automatically after any command that changes the data. There is no need to save manually.
-
-### Editing the data file
```
Comment (wei2912) [inline docs/UserGuide.md:161]: Add this section

**Review by luffingluffy — APPROVED** [review]
LGTM!

**Pull request merged**